daemon/algod/api/server/router.go

```diff
@@ -71,13 +71,13 @@ import (
 	"github.com/algorand/go-algorand/daemon/algod/api/server/lib/middlewares"
 	"github.com/algorand/go-algorand/daemon/algod/api/server/v1/routes"
 	"github.com/algorand/go-algorand/daemon/algod/api/server/v2"
-	"github.com/algorand/go-algorand/daemon/algod/api/server/v2/generated"
 	"github.com/algorand/go-algorand/logging"
 	"github.com/algorand/go-algorand/node"
+	"github.com/algorand/go-algorand/util/tokens"
 )
 
 const (
-	apiV1Tag = "v1"
+	apiV1Tag = "/v1"
```

Comment:
It's a bit weird that the tag has a `/` in it

Reply:
yes, I did it to ensure the registration with Echo is symmetrical between the v1 and v2 entry points. I can eliminate it here, but need to make corresponding changes to the code generator ( which would be good, but abit outside the scope of this task )